stable/prometheus/templates/kube-state-metrics-clusterrole.yaml

```diff
@@ -1,4 +1,5 @@
-{{- if and .Values.kubeStateMetrics.enabled .Values.rbac.create -}}
+{{- if .Values.kubeStateMetrics.enabled -}}
+{{- if .Values.rbac.create }}
```

Comment:
@schoofsc 
Why you break up 'and' condition to multiple line for?

Reply:
@zanhsieh I wanted everything to be consistent.